utils/hct/hctdb.py

```diff
@@ -389,28 +451,55 @@ def build_indices(self):
 
     def build_opcode_enum(self):
         # Build enumeration from instructions
-        OpCodeEnum = db_dxil_enum(
-            "OpCode", "Enumeration for operations specified by DXIL"
-        )
+        OpCodeEnum = self.get_dxil_op_table().op_enum
```

Comment:
We're very subtly relying on the fact that `self.get_dxil_op_table()` can only ever be the `CoreOps` table at this point, which is pretty confusing. The only real hint to what's going on here is the `# Skip CoreOps table` comment in the loop over the rest of the tables below.

This should probably grab `self.dxil_op_tables[0]` or `self.dxil_op_tables_by_name['CoreOps']` instead.